lib/src/test/java/com/auth0/android/lock/OptionsTest.java

```diff
@@ -94,23 +107,20 @@ public void shouldNotSetPrivacyPolicyURLWhenInvalidURL() {
     }
 
     @Test
-    public void shouldSetTermsOfServiceURL() {
-        Options options = new Options();
-        options.setAccount(auth0);
+    public void shouldSetTermsOfServiceURL() throws Exception {
         options.setTermsURL("https://valid.url/terms");
 
         Parcel parcel = Parcel.obtain();
         options.writeToParcel(parcel, 0);
         parcel.setDataPosition(0);
 
         Options parceledOptions = Options.CREATOR.createFromParcel(parcel);
+        assertThat(options.getTermsURL(), is(notNullValue()));
         assertThat(options.getTermsURL(), is(equalTo(parceledOptions.getTermsURL())));
     }
 
     @Test
-    public void shouldNotSetTermsOfServiceURLWhenInvalidURL() {
-        Options options = new Options();
-        options.setAccount(auth0);
+    public void shouldNotSetTermsOfServiceURLWhenInvalidURL() throws Exception {
         options.setTermsURL("an-invalid/url");
```

Comment:
If the given url it's not valid, the it will default to Auth0's terms/policy urls. I was trying to avoid throwing an exception at this point, and just warning the user that the default will be used on the LogCat. Anyway, I can add that change.